.gitignore

```diff
@@ -18,3 +18,4 @@ gen/
 coverage/
 .rpt2_cache
 vis-network-*.tgz
+/dist/
```

Comment:
I think we talked about this earlier. I'm not sure if we simply can remove the `dist`folder because apps may depend on it.

Reply:
We are not removing it, we are moving it to gh-pages branch and publishing that instead. Nothing changes from the outside we'll just switch GitHub Pages from master to a branch.